### ISSUE TITLE
Allow slog.Group() when setting no-mixed-args: true or kv-only: true 

### ISSUE THREAD

**Issue by cfiderer** [go-simpler/sloglint#50]
The `slog.Group()` function is the only means to put some grouped data elements between other elements into the log.

A call like

`logger.Info("message", "key1", value1, slog.Group("group", "key2", value2, "key3", value3), "key4", value4)`

should be allowed, and the arguments to `slog.Group()` should be checked for key-value pairs, too.

**Comment by tmzane** [comment]
I completely forgot that `slog.Group()` existed. I agree, it should not be reported.

**Comment by ravicious** [comment]
This can be used as a workaround:

```go
logger.Info("message", slog.String("key1", value1), slog.Group("group", "key2", value2, "key3", value3), slog.Int("key4", value4))
```
